src/signals/incident-management/containers/ReporterContainer/useReporter.ts

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+import { showGlobalNotification } from 'containers/App/actions'
+import { TYPE_LOCAL, VARIANT_ERROR } from 'containers/Notification/constants'
+import useFetch from 'hooks/useFetch'
+import { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { useParams } from 'react-router-dom'
+import configuration from 'shared/services/configuration/configuration'
+import { Incident, Reporter } from './types'
+
+const PAGE_SIZE = 10
+
+export interface ReporterHook {
+  selectedIncident?: Incident
+  selectedIncidentId?: number
+  setSelectedIncidentId: (id: number) => void
+  isLoading: boolean
+  reporter: {
+    originalIncidentId: string
+    email?: string
+    incidents?: Reporter
+  }
+}
+
+export const useReporter = (): ReporterHook => {
+  const storeDispatch = useDispatch()
+  const [selectedIncidentId, setSelectedIncidentId] = useState<number>()
+  const { id } = useParams<{ id: string }>()
```

Comment:
I would move this to the parent and pass it as parameter here. Is difficult in the container to figure out where the reporter  comes from